src/react/useDocuments.ts

```diff
@@ -2,11 +2,11 @@
 
 import { useEffect, useState } from 'react';
 import type { AppBridgeTheme } from '../AppBridgeTheme';
-import { FrontifyDocumentGroup } from '../types';
-import type { FrontifyDocument } from '../types/FrontifyDocument';
+import { DocumentGroup } from '../types';
```

Comment:
This is also a type so we could have:
```suggestion
import type { DocumentGroup } from '../types';
```